### DISCUSSION TITLE
Support for nested tags

### DISCUSSION THREAD

**Discussion by andrzej-regulski** [st3v3nmw/obsidian-spaced-repetition#384] [Q&A]
This plugin is awesome, thanks for the effort!

I am huge fan of nested/hierarchical tags in Obsidian and apparently the plugin does not support these. 

For instance, if I direct Flashcards Tags to `#type/flashcards` the plugin recognizes this as `#type` this is kind of useless in my setup as I categorize all of my notes using a set of categories nested under `#type`. It would be awesome to get this fixed.

**Comment by sameerad2001** [comment]
This plugin does support nested tags
Format : `#flashcards/topic/subTopic`

![image](https://user-images.githubusercontent.com/66942584/151109203-f96a7508-7c4c-4c6a-976d-4808116e9f2e.png)

hope this helps
